types/js-cookie/js-cookie-tests.ts

```diff
@@ -41,11 +40,11 @@ Cookies.defaults.path = '';
 delete Cookies.defaults.path;
 
 const PHPCookies = Cookies.withConverter({
-    write(value) {
-        return encodeURIComponent(value)
+    write(value: string | object) {
```

Comment:
Not a big thing, but it's actually better to check it like this:
```typescript
write(value) {
  value; // $ExpectType string | object
  // ...
}
```
The reasoning is that this way you verify that the interface doesn't do something like `write(value: any) => void`, which you would miss with just a type signatute for the `write` method.